### DISCUSSION TITLE
Avoid to override existing Changie config file if already exists

### DISCUSSION THREAD

**Discussion by llaville** [miniscruff/changie#403] [Ideas] [answered]
With Changie v1.10.0

When you have a `.changie.yaml` file in your repository and run the `changie init` command, the default version of `.changie.yaml` override the current existing version.

Consider either to : 
1.  add an option to avoid this behavior 
1.  display a warning message 
1.  `changie init` write a `.changie.yaml.dist` file instead

PS: It's same for `.changes/` folder and header template file

**Comment by miniscruff** [comment, marked as answer]
Hmm correct, `init` does not make any attempt to not clobber existing files or folders. I am not sure how needed this feature is ( I mean with git you would probably just undo any changes and move on ) but I think not overriding existing values is a good idea. I would consider it a bug that running init again would override values. 

My proposal would be:
1. Init should check for a `.changie.yaml` and do nothing if a file is found, it should instead return an error and show a message
2. A new `--force` option should be added to bypass this check

#404 was created to address the proposal

**Reply by llaville** [reply]
Don't forget template files too (header / footer)

**Reply by miniscruff** [reply]
Right, if a `.changie.yaml` file is found, it won't do anything. If one is not found, it will do the normal process which may or may not override files or folders at that point. The assumption is, no changie config file, not yet initialized so do everything. We would assume you meant to override files or folders at that point.
